### ISSUE TITLE
Enhancement: test if a different native view works to create snackbar as default

### ISSUE THREAD

**Issue by jscti** [nstudio/nativescript-snackbar#10]
Hi there, I got an error using this lib.

I'm trying to show a snackbar in the ngOnInit of my AppComponent, and this with the sample code, nothing more :

```
export class AppComponent implements OnInit {
    ngOnInit() {
        const snackbar = new SnackBar();

        snackbar.simple('Snackbar', 'red', '#fff', 3, false).then((args) => {
            console.log('dismissed')
        })
    }
}
```

And got the error : 

> JS: TypeError: Cannot read property 'android' of null
> JS:     at file:///node_modules/@nstudio/nativescript-snackbar/snackbar.js:33:0
> JS:     at new ZoneAwarePromise (file:///node_modules/@nativescript/angular/zone-js/dist/zone-nativescript.js:902:0)
> JS:     at SnackBar.push.../node_modules/@nstudio/nativescript-snackbar/snackbar.js.SnackBar.simple (file:///node_modules/@nstudio/nativescript-snackbar/snackbar.js:27:0)
> JS:     at 

TNS : 6.2.2
TNS Android : 6.2.0
@nstudio/nativescript-snackbar: 1.1.1

**Comment by jscti** [comment]
Nevermind : https://github.com/bradmartin/nativescript-snackbar/issues/36

**Comment by bradmartin** [comment]
Yea just a slight timing/lifecycle issue there. Glad you found the answer :)

**Comment by jscti** [comment]
Yep but .. It's still a problem because we can't rely on this lib to handle errors for example .. 
Because an error can occur anytime, before UI is ready.

So in this case I fall back on tns-core-module `alert` which has not this bug .. but .. beh

**Comment by bradmartin** [comment]
Understand, there might be a way around it by dropping the `frame` approach and using a more native approach to grab the window. I'd have to refer to the documentation to see what is a valid argument to pass to the snackbar, if the window view is acceptable, then we could change the default to that if the `currentPage.frame` isn't available.

**Comment by bradmartin** [comment]
Fixed with `1.1.2`
